scipy/stats/tests/test_stats.py

```diff
@@ -7201,10 +7207,14 @@ def test_same_distribution(self):
             stats.wasserstein_distance([1, 1, 1, 4], [4, 1],
                                        [1, 1, 1, 1], [1, 3]),
             0)
+        assert_equal(stats.wasserstein_distance(
+            [[0, 0], [15, 0.2]], [[15, 0.2], [0, 0]]), 0)
+        assert_equal(stats.wasserstein_distance(
```

Comment:
Here's the failure. Of course, they are essentially equal, with an error of `1.0839226509442299e-17`. If it's not something that can be fixed, use `assert_allclose(result, 0, atol=1e-16)`.

Reply:
Sure I would look at the lint tools, but I assume it's also okay if i manually change all the lint failures?

Reply:
You _should_ manually fix the failures. The lint tools will just help you check whether you actually fixed them all.